src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
```

Comment:
Connect fonts to your app, use google fonts. You have Roboto fonts on your PC but the test environment does not, and maybe other users have not installed fonts on their PC. It should fix some tests